lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -183,8 +183,8 @@ function dosomething_northstar_build_ns_user($user, $form_state) {
   }
   foreach ($address as $ns_key => $drupal_key) {
     $field = $user->field_address[LANGUAGE_NONE][0];
-    if (isset($field[$drupal_key]['value'])) {
-      $ns_user[$ns_key] = $field[$drupal_key]['value'];
+    if (isset($field[$drupal_key])) {
+      $ns_user[$ns_key] = $field[$drupal_key];
```

Comment:
ahh I see what you mean now, thanks for clarifying, @angaither ! I was only testing against creating a new user, where they don't ask for the address fields. Just tested on updating the profile and adding in the address and you're right - it doesn't send over the address fields since I deleted `value`. I'll work on updating this now!